model/src/main/kotlin/PackageCurationData.kt

```diff
@@ -89,33 +89,33 @@ data class PackageCurationData(
      * @return The curated package.
      */
     fun apply(base: CuratedPackage): CuratedPackage {
-        val pkg = base.pkg
+        val basePkg = base.pkg
 
-        val curatedVcs = if (vcs != null) {
+        val resultVcs = if (vcs != null) {
```

Comment:
Why is `result` clearer than `curated`? Same for `resultPackage` below.

Reply:
Because both, the input (parameter) and the output are curated packages.

Reply:
Can we keep this change as-is?

Reply:
How about naming it `derivedVcs`, `derivedPackage`, `derivedCurations`.

Reply:
I've changed the approach a bit.
Does it resolve the issue?